app/gimletlet/app.toml

```diff
@@ -259,3 +259,21 @@ pins = [
     # SPI6 CS repurposed for debugging
     { name = "DEBUG", pin = { port = "G", pin = 8, af = 0, direction = "output"}}
 ]
+
+[[config.i2c.devices]]
+device = "lm5066"
+controller = 4
+address = 0x16
+description = "LM5066 evaluation board"
+power = { rails = [ "LM5066_EVL_VOUT" ] }
+sensors = { voltage = 1, current = 1, temperature = 1 }
+
+[tasks.power]
+name = "task-power"
+priority = 6
+max-sizes = {flash = 65536, ram = 16384 }
+stacksize = 3800
+start = true
+task-slots = ["i2c_driver", "sensor"]
+notifications = ["timer", "external_badness"]
```

Comment:
... `"external_badness"`?

Reply:
Stowaway turd in the Gimlet app definition from 4f29df05b455490636ed67a87940ab77a518986f and then copy-pasta'd here; removed both.